_docs/style/text-conventions/text-conventions-landing-page.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Text conventions
+layout: categories
+category: Style
+usage: >
+  Use these user interface (UI) text conventions if you are a designer, developer, or contributor of content in Rackspace UIs. These UI text guidelines ensure that a Rackspace UI is usable, consistent, and concise.
+status: stable
+resource: true
+last-modified: 2018-06-06
+---
+
+- [Alerts and messages]({{site.baseurl}}/style/alert-messages.html)
```

Comment:
remove the hard-coded list, and paste this code:

```
{% for article in site.docs %}{% if article.parent == 'Text Conventions' %}
- [{{article.title}}]({{site.baseurl}}{{article.url}}){% endif %}{% endfor %}
```
This relies on the parent attribute of each text article being Text Conventions.  So, the next few comments will be about updating the front matter on each article.